packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -71,4 +78,48 @@ export class PluginsPage {
 		await this.page.click( selectors.breadcrumb( text ) );
 		assert.match( this.page.url(), /.*\/plugins$/ );
 	}
+
+	/**
+	 * Click Monthly Pricing
+	 */
+	async clickMonthlyPricing(): Promise< void > {
+		await this.page.click( selectors.monthlyPricingToggle );
+	}
+
+	/**
+	 * Click Annual Pricing
+	 */
+	async clickAnnualPricing(): Promise< void > {
+		await this.page.click( selectors.annualPricingToggle );
+	}
+
+	/**
+	 * Select monthly
+	 */
+	async selectMonthlyPricing(): Promise< void > {
+		await this.page.click( selectors.pricingToggle );
+		await this.page.click( selectors.monthlyPricingSelect );
```

Comment:
@dpasque couldn't figure this one out, any ideas as to how best to achieve this?

Trying to toggle the `/plugins` screen's pricing like so:

![Screenshot 2022-02-21 at 14-09-06 Plugins ‹ Site Title — WordPress com](https://user-images.githubusercontent.com/811776/154882791-4fbe0f9f-95ad-493a-8cad-7f9355dcc896.png)

(note tests see this smaller screen version, wide screens see a link based toggle - is there prior art around loading a page with a larger browser resolution I could use to test both views?)

Reply:
>is there prior art around loading a page with a larger browser resolution I could use to test both views?
Ah, interesting! So right now the tests are run at two different viewport sizes, a mobile and a desktop, and it looks like our current desktop one is too small!

I know @worldomonation did a lot of looking into device usage and common breakpoints when picking our two resolutions. He is probably the best person to weigh in on potentially adding a wide screen view!

Reply:
I haven't gone a looked at the underlying code or unit test coverage, but one other suggestion I have is that you may be able to still get sufficient coverage without including E2E tests for the widescreen toggle by combining these existing tests with good unit test coverage.

For example, if you can have unit test coverage that validates that click events on the widescreen toggle in a virtual DOM are respected and make the correct state updates, and then you have these E2E tests that use the drop down selection and and demonstrate that the UI of the entire page updates in real time to reflect the selection, I think you're golden! Because you've proved that...

1. Both control components update the state correctly
2. Updates to the state are correctly reflected in the broader page and flow where these components are located.

Just an idea! 🙂

Reply:
Thanks re screen sizes. I think sticking with the smaller viewport is fine, I'm guessing that's just what is most common.

Would you be able to take a look at why this test is currently failing? I couldn't figure out how to make it click / find the right element, it does some kind of scrolling up and down the page but never finds the dropdown on the page.

Reply:
Figured this out :tada: